_episodes/03-github-pages.md

```diff
@@ -1,21 +1,160 @@
 ---
-title: "Hosting Pages on GitHub"
+title: "Hosting Pages and Websites on GitHub"
 teaching: 0
 exercises: 0
 questions:
-- "How do I publish my pages via GitHub?"
+- "How do I publish my page or a website via GitHub?"
```

Comment:
Should we add another objective such as "How do I configure my website?"

Reply:
Good idea @annefou. Fixed in 28b5cdd.